core/block_validator_test.go

```diff
@@ -35,7 +35,7 @@ func testChainConfig() *ChainConfig {
 		Forks: []*Fork{
 			&Fork{
 				Name:  "Homestead",
-				Block: big.NewInt(1150000),
+				Block: big.NewInt(0),
```

Comment:
Block 1150000 is when Homestead was initiated.  We should probably keep that as the Homestead fork starting block